sql/moz-fx-data-shared-prod/udf_js/convert_path_to_column_name/udf.sql

```diff
@@ -0,0 +1,37 @@
+/*
+Converts the `path` column in `monitoring.telemetry_missing_columns` to a format in 'INFORMATION_SCHEMA'. 
+This is to check if the missing column has been added to the table
+
+See:
+- https://bugzilla.mozilla.org/show_bug.cgi?id=1823724
+for more context
+
+*/
+
+CREATE OR REPLACE FUNCTION udf_js.convert_path_to_column_name(input STRING)
+RETURNS STRING
+LANGUAGE js
+AS r"""
+    probe_name = input.split('`').map((el, i) => i % 2 ? '`' + el + '`' : el).filter((el) => el != '.');
+    result = [];
+    var key_path_str ='';
+    for (var i = 0; i < probe_name.length; i++) {
+            if (i == 3){
```

Comment:
Will this always be true for the 3rd iteration of the loop?

Reply:
if it is not there, the if clause will be skipped